src/DecryptionFailureTracker.ts

```diff
@@ -168,30 +180,43 @@ export class DecryptionFailureTracker {
     //     localStorage.setItem('mx-decryption-failure-event-ids', JSON.stringify([...this.trackedEvents]));
     // }
 
+    /** Callback for when an event is decrypted.
+     *
+     * This function should be called after a decryption attempt on an event.  It
+     * should be called whether the decryption is successful or not.
+     *
+     * @param matrixEvent the event that was decrypted
+     *
+     * @param nowTs the current timestamp
+     */
     public eventDecrypted(e: MatrixEvent, nowTs: number): void {
```

Comment:
this should probably not be public any more?

Reply:
On the one hand, I agree.  On the other hand, I'm not particularly excited about littering `DecryptionFailureTracker-test.ts` with a ton of `@ts-ignore`s